shotover-proxy/tests/helpers/mod.rs

```diff
@@ -139,6 +141,32 @@ impl ShotoverManager {
         .unwrap()
     }
 
+    #[allow(unused)]
+    pub fn cassandra_connection_tls(
```

Comment:
Shouldnt this go next to the `fn cassandra_connection` ?

Reply:
I've moved `fn cassandra_connection` to `ShotoverManger` in #512, so once that is merged they will be in the same place.

Reply:
ah, alright, yeah I think its fine for both of them to go in ShotoverManager.
While its true that we could use cassandra_connection to connect directly to cassandra instead of through shotover, there is no case where we would want to do that without also having a shotover instance running anyway.